cityofaustin/atd-moped
# Update related phase milestone

## Associated issues

none, this is a PR branching off of https://github.com/cityofaustin/atd-moped/pull/1462


## Testing
**URL to test:** 
local since the target branch is not main

**Steps to test:**

In the milestones table, open the milestones autocomplete input and try selecting different milestones. If there is a related phase, you will see the field update. 

Try tabbing back and forth through the table. 

**Questions**

Does this feel too hacky? I had to define the related phase id field as editable in order to update it during the Edit Mode. I tried setting the RenderEditCell to be a simple div or span, but the component needs to be an input for the focus to anchor onto. 

But then since the input is disabled, the focus was getting stuck. I tried  removing the field from the array of tab-able columns (https://github.com/mui/mui-x/issues/11999), but the focus would still get stuck and not successfully skip over to the date/time inputs. 

---
#### Ship list
   - Added "Update the Milestone type, and you should see the related phase update in the form before saving."
